ynr/apps/candidates/templates/candidates/generic-merge-error.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+{% block content %}
+  {% if error_message %}
+  <h1><a href="{{ person.get_absolute_url }}">{{ person.name }}</a></h1>
+  <h2>Error Merging</h2>
+  <p>Sorry, something when wrong when merging. The error message was:</p>
+  <p><code>{{ error_message }}</code></p>
+  {% endif %}
+
+  <form class="person__actions__action person__actions__merge" id="person-merge" action="{% url 'person-merge' person_id=person.id %}" method="post">
```

Comment:
Is this `<form> identical to the one in `person-edit.html`? If so, could we pull it out into an include that can be shared by `generic-merge-error.html` and `person-edit.html`?

Reply:
I did consider this, but it's just different enough to make it worthwhile, IMHO:

1. The `person-edit.html` form doesn't have the error messages, and in fact has no ability to validate the form and show errors on it

2. The `person-edit.html` doesn't know the other person ID, and meaning there'd be a missing template var

3. We might want to iterate this page to show other information about the merge, meaning the form might change.

4. This is a full sized form when the other one is in a side bar

Point taken about DRY, but I think in this case I'm happier with a bit of duplication